### ISSUE TITLE
Add Contrast Correction to Cov 2D

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#918]
Add amplitude/contrast correction method from Yunpeng to cov 2d code. 

https://github.com/yunpeng-shi/fast-cryoEM-PCA/blob/main/fast_cryo_pca.py#L862

Ostensibly, there are two distinct outputs we probably want to expose. i) just the correction factors (ie preprocessing).  ii) apply to denoising.

Yunpeng reported there was some small improvement over the existing cov2d.  Based on his description of the method I'm inclined to believe this would be a useful addition that we should make time for after the larger basis project.

**Comment by janden** [comment]
Agreed, but it's definitely in the *nice-to-have* category for now.
